_docs/7_SubsquidArchive/docker-compose.yml

```diff
@@ -9,20 +9,29 @@ services:
       POSTGRES_USER: postgres
       POSTGRES_PASSWORD: postgres
       POSTGRES_DB: squid-archive
+    ports:
+      - "0.0.0.0:5533:5533"
 
   ingest:
     depends_on:
       - db
     restart: on-failure
     image: subsquid/substrate-ingest:firesquid
-    command: [
-      "-e", "ws://node:9944",
-      "-c", "10",
-       "--prom-port", "9090",
-       "--out", "postgres://postgres:postgres@db:5432/squid-archive"
-    ]
+    command:
+      [
+        "-e",
+        "ws://node:9945",
+        "-c",
+        "10",
+        "--prom-port",
+        "9191",
+        "--out",
+        "postgres://postgres:postgres@db:5432/squid-archive",
+      ]
     environment:
       NODE_TLS_REJECT_UNAUTHORIZED: 0
+    ports:
+      - "0.0.0.0:9191:9191"
```

Comment:
These dont need to be exposed on the host on all IPs either since they will be proxied via Nginx.